package.json

```diff
@@ -31,17 +31,25 @@
   },
   "devDependencies": {
     "@types/config": "^3.3.0",
+    "@types/knex": "^0.16.1",
     "@types/node": "^18.0.0",
+    "@types/sqlite3": "^3.1.8",
     "@types/tap": "^15.0.5",
     "@typescript-eslint/eslint-plugin": "^5.61.0",
     "@typescript-eslint/parser": "^5.61.0",
     "concurrently": "^7.0.0",
     "eslint": "^8.44.0",
     "eslint-config-prettier": "^8.8.0",
     "fastify-tsconfig": "^1.0.1",
+    "jest": "^29.6.2",
+    "knex": "^2.5.1",
```

Comment:
Are all the newly added packages neccessery?
`mocha`, `jest`, etc.

Reply:
those are removed.